rct229/rulesets/ashrae9012019/section19/section19rule20.py

```diff
@@ -0,0 +1,199 @@
+                * percent_of_total_return_p,
+                "expected_baseline_fan_power_exhaust": total_modeled_fan_power_b
+                * percent_of_total_exhaust_p,
+                "expected_baseline_fan_power_relief": total_modeled_fan_power_b
+                * percent_of_total_relief_p,
+            }
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            hvac_sys_total_supply_fan_power_b = calc_vals[
+                "hvac_sys_total_supply_fan_power_b"
+            ]
+            hvac_sys_total_return_fan_power_b = calc_vals[
+                "hvac_sys_total_return_fan_power_b"
+            ]
+            hvac_sys_total_relief_fan_power_b = calc_vals[
+                "hvac_sys_total_relief_fan_power_b"
+            ]
+            expected_baseline_fan_power_supply = calc_vals[
+                "expected_baseline_fan_power_supply"
+            ]
+            expected_baseline_fan_power_return = calc_vals[
+                "expected_baseline_fan_power_return"
+            ]
+            expected_baseline_fan_power_relief = calc_vals[
+                "expected_baseline_fan_power_relief"
+            ]
+            return (
+                std_equal(
+                    hvac_sys_total_supply_fan_power_b,
+                    expected_baseline_fan_power_supply,
+                )
+                and std_equal(
+                    hvac_sys_total_return_fan_power_b,
+                    expected_baseline_fan_power_return,
+                )
+                and std_equal(
+                    hvac_sys_total_exhaust_fan_power_b,
+                    expected_baseline_fan_power_exhaust,
+                )
+                and std_equal(
+                    hvac_sys_total_relief_fan_power_b,
+                    expected_baseline_fan_power_relief,
+                )
+            )
+
+        def get_fail_msg(self, context, calc_vals=None, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+            expected_baseline_fan_power_supply = calc_vals[
+                "expected_baseline_fan_power_supply"
+            ]
+            expected_baseline_fan_power_return = calc_vals[
+                "expected_baseline_fan_power_return"
+            ]
+            expected_baseline_fan_power_exhaust = calc_vals[
+                "expected_baseline_fan_power_exhaust"
+            ]
+            expected_baseline_fan_power_relief = calc_vals[
+                "expected_baseline_fan_power_relief"
+            ]
+
+            return f"The calculated system fan power does appear to be distributed to the supply, return, exhaust, and relief fans in the same proportion as the proposed design for {hvac_id_b}.The expected modeled baseline supply, return, exhaust, and relief kW is {expected_baseline_fan_power_supply}, {expected_baseline_fan_power_return}, {expected_baseline_fan_power_exhaust}, {expected_baseline_fan_power_relief} respectively."
```

Comment:
Shouldnt this be "The calculated system fan power does **not** appear to be distributed..."?

Reply:
Good catch! Addressed.